src/humancellatlas/data/metadata/helpers/schema_validation.py

```diff
@@ -0,0 +1,40 @@
+from functools import (
+    lru_cache,
+)
+import json
+import logging
+
+from jsonschema import (
+    FormatChecker,
+    ValidationError,
+    validate,
+)
+import requests
+
+from humancellatlas.data.metadata.api import (
+    JSON,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class SchemaValidator:
+
+    @classmethod
+    def validate_json(cls, file_json: JSON, file_name: str):
+        try:
+            schema = cls._download_schema(file_json['describedBy'])
+        except json.decoder.JSONDecodeError as e:
+            schema_url = file_json['describedBy']
+            raise Exception('Failed to parse schema JSON', file_name, schema_url) from e
+        try:
+            validate(file_json, schema, format_checker=FormatChecker())
+        except ValidationError as e:
+            raise ValidationError(f'File {file_name}') from e
```

Comment:
I read your explanation regarding ValidationError not supporting multiple arguments. But chaining one ValidationError instance to another like this is a bit unorthodox for exceptions raised by libraries.

```suggestion
            raise RequirementError(*e.args, file_name) from e
```

The reason I want to avoid string interpolation for exception messages is because it adds noise to the code and circumvents the pattern that Exception establishes. There is a reason why the Exception constructor has a `*args` constructor argument.

The interpolation would have needed `!r` anyways. Using `*args` is much less fiddly and `repr` is applied automatically.